### ISSUE TITLE
Use recommended kafka config in java-customer-registry-kafka-quickstart docker-compose.yml

### ISSUE THREAD

**Issue by raboof** [lightbend/akkaserverless-java-sdk#851]
In the [docs](https://developer.lightbend.com/docs/akka-serverless/projects/message-brokers.html#_use_a_kafka_broker_locally) we show how to create a `docker-compose.yml` that connects to Kafka, but our [customer-registry-sample](https://github.com/lightbend/akkaserverless-java-sdk/blob/main/samples/java-customer-registry-kafka-quickstart/docker-compose.yml) application still connects to a Google PubSub emulator.

Probably because we still use that for testing ourselves?

**Comment by beritou** [comment]
I agree, `java-customer-registry-kafka` should demonstrate how to use Kafka locally.

Testing the Google PubSub emulator is a separate concern and is ideally handled elsewhere.

**Comment by beritou** [comment]
I see this is already underway in https://github.com/lightbend/akkaserverless-java-sdk/pull/852
